views/scss/qti/_graphic-gap.scss

```
@@ -40,6 +40,75 @@
         }
     }
 
+    &.qti-choices-top {
+        ul.source {
+            position: absolute;
+            top: -78px;
+            left: 0;
+            z-index: 1000;
+            background: whiten($uiClickableDefaultBg, .5);
+            @include simple-border();
```

Stylelint (17.13.0)

[error] Unexpected parentheses in argumentless mixin "simple-border" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)